docs/general/administration/hardware-acceleration/amd.md

```diff
@@ -435,7 +429,7 @@ Root permission is required.
          image: jellyfin/jellyfin
          user: 1000:1000
          group_add:
-           - "122" # Change this to match your "render" host group id and remove this comment
+           - '122' # Change this to match your "render" host group id and remove this comment
```

Comment:
I'm not sure about all the double quote to single quote changes in yaml blocks because they do have different meanings. Although the changes looks okay and should still work to me but I'm not sure if those changes are necessary